<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileOverwriteDialog</class>
 <widget class="QDialog" name="FileOverwriteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>The following file already exists on disk:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="pathLabel">
     <property name="text">
      <string>&lt;b&gt;bitstream name here&lt;/b&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Would you like to use the exisiting file, or overwrite it?</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="useExistingButton">
       <property name="text">
        <string>Use Exisiting File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="overwriteButton">
       <property name="text">
        <string>Overwrite Existing File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="applyToAllCheckbox">
     <property name="text">
      <string>Do this for all conflicts</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
